charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
```

Comment:
IMO the "scope" section should be about the kind of work that the team can take on, not about the details of which proposals are expected to be accepted.

I'd suggest the scope as some thing like:

```
The output of the Interop project is metrics designed to measure and improve the interoperability of the web platform.

These metrics are expected to be comprised of:
* Groups of automated tests that can be used to automatically compute an interoperability score for specific platform features in implementations.
* Work to improve the interoperability of the platform that cannot be measured through test pass rates and is scored by consensus decision.

The Interop group takes ownership of the existing Compat-2021 and Interop-2022 projects, and those will form a basis for future work.

This is not a venue for standardisation, and work on new platform features is out of scope. However the group may liase with existing standards groups to help progress work which affects the interoperability of the platform, or the ability to test and measure interopability of existing features.
```

All the existing content about which proposals are actually likely to be accepted is really a consequence of the "proposals require consensus" rule, and therefore shouldn't form an independent part of the charter. Instead we should add some informative text to a proposer-facing README about the kinds of proposals that are likely to find consensus (i.e. those which are in a recognised standards group, are aimed at browsers, where all implementors are positive on the feature design, and where the level of stability of the specification is appropriate for implemention work to occur).

This has the advantage that we don't need to make charter-level changes if specific participants change their opinions about which kinds of proposals they are unwilling to accept.

Reply:
I've applied the suggestion with only minor edits/adaptations.